shared/src/main/scala/pl/metastack/metarx/reactive/stream/Is.scala

```diff
@@ -9,7 +9,13 @@ import pl.metastack.metarx.ReadChannel
 trait Is[T] {
   /** Current value is equal to `value` */
   def is(value: T): ReadChannel[Boolean]
+  def is[U](value: ReadChannel[U]): ReadChannel[Boolean]
+  def ===(value: T) = is(value)
+  def ===[U](value: ReadChannel[U]) = is(value)
```

Comment:
You are of course correct. == between different types likely mostly result in in false result. But I'm still suspecting that T should have variance + (?) to allow for comparing types in a hierarchy that supports it.